build-runtime.sh.in

```diff
@@ -56,24 +67,12 @@ HEXLENGTH=$(objdump -h runtime | grep .upd_info | awk '{print $3}')
 dd bs=1 if=runtime skip=$(($(echo 0x$HEXOFFSET)+0)) count=$(($(echo 0x$HEXLENGTH)+0)) | xxd
 
 # Insert AppImage magic bytes
-
 printf '\x41\x49\x02' | dd of=runtime bs=1 seek=8 count=3 conv=notrunc
 
 # Convert runtime into a data object that can be embedded into appimagetool
-
 ld -r -b binary -o data.o runtime
 
```

Comment:
What is wrong about the tests in the following lines? I'd like to keep them for reference if possible. If not possible, I'd at least like to keep them commented out.

Reply:
The tests are performed just a few lines above as well. I didn't see a reason to call them twice.